Changes to reflect the deprecation of pod-manifest-path argument (#44266)
kubernetes/website: content/en/docs/tasks/configure-pod-container/static-pod.md

   EOF
   ```

> 1. Configure the kubelet on that node to set a `staticPodPath` value in the
>    [kubelet configuration file](/docs/reference/config-api/kubelet-config.v1beta1/).   
>    See [Set Kubelet Parameters Via A Configuration File
>    ](/docs/tasks/administer-cluster/kubelet-config-file/) for more information.
> 
>    An alternative and deprecated method is to configure the kubelet on that node to look for static Pod manifests locally, using a command line argument. To use the deprecated approach, start the kubelet with the `--pod-manifest-path=/etc/kubernetes/manifests/` argument.
>       

Reviewer: A minor nit for improved readability, maintainability(especially for localization), and ease of review: consider wrapping the long text like the rest of the content in the file and avoid splitting the markdown link across two different lines.

```suggestion
1. Configure the kubelet on that node to set a `staticPodPath` value in the
   [kubelet configuration file](/docs/reference/config-api/kubelet-config.v1beta1/). 
   See [Set Kubelet Parameters Via A Configuration File](/docs/tasks/administer-cluster/kubelet-config-file/)
   for more information.

   An alternative and deprecated method is to configure the kubelet on that node
   to look for static Pod manifests locally, using a command line argument.
   To use the deprecated approach, start the kubelet with the
   `--pod-manifest-path=/etc/kubernetes/manifests/` argument.
      
```

   EOF
   ```

1. Configure the kubelet on that node to set a `staticPodPath` value in the
   [kubelet configuration file](/docs/reference/config-api/kubelet-config.v1beta1/).  
   See [Set Kubelet Parameters Via A Configuration File](/docs/tasks/administer-cluster/kubelet-config-file/)
   for more information.

>    An alternative and deprecated method is to configure the kubelet on that node
>    to look for static Pod manifests locally, using a command line argument.
>    To use the deprecated approach, start the kubelet with the  

Reviewer 2: I think this paragraph can be improved. I'm ok to let this PR merge and this can be iterated on

```suggestion
   An alternative and deprecated method is to configure the kubelet on that node
   to look for static Pod manifests locally, using a command line argument.
   To use the deprecated approach, start the kubelet with the  
```

   EOF
   ```

> 1. Configure your kubelet on the node to add the `staticPodPath: <the directory>` field in the

Author: Thanks for the suggestion.

   [kubelet configuration file](/docs/reference/config-api/kubelet-config.v1beta1/).   
   See [Set Kubelet Parameters Via A Configuration File
   ](/docs/tasks/administer-cluster/kubelet-config-file/) for more information.

>    {{< note >}}

Author: Thanks for the suggestion.

>    Alternately, a __deprecated__ option is to configure your kubelet on the node to use this directory by running it with

Reviewer 3: ```suggestion
   An alternative and deprecated method is to configure the kubelet on that node to look for static Pod manifests
   locally, using a command line argument. To use the deprecated approach, start the kubelet with the
```

Author: Thanks for the suggestion.

   `--pod-manifest-path=/etc/kubernetes/manifests/` argument.
   On Fedora, edit `/etc/kubernetes/kubelet` to include this line:

   ```

>    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --pod-manifest-path=/etc/kubernetes/manifests/"

Author: Thanks for the suggestion.

>    {{< /note >}}

Author: Thanks for the suggestion.
